### ISSUE TITLE
test(ethereum): additional properties to deployContract does not throw error

### ISSUE THREAD

**Issue by ashnashahgrover** [hyperledger/cacti#3507]
**Describe the bug**

The deployContract function in cactus-plugin-ledger-connector-ethereum takes DeployContractV1Request as an argument, which already declares additional properties as forbidden in the Open API spec as can be seen [here](https://github.com/hyperledger/cacti/blob/957da7c3e1d80068391485a825ba6bb1e68333ac/packages/cactus-plugin-ledger-connector-ethereum/src/main/json/openapi.json#L435): 

```
"DeployContractV1Request": {
        "type": "object",
        "required": ["web3SigningCredential", "contract"],
        **"additionalProperties": false,**
        "properties": {
          "web3SigningCredential": {
            "$ref": "#/components/schemas/Web3SigningCredential"
          },
          "contract": {
            "oneOf": [
              {
                "$ref": "#/components/schemas/ContractJsonDefinition"
              },
              {
                "$ref": "#/components/schemas/ContractKeychainDefinition"
              }
            ],
            "nullable": false
          },
          "constructorArgs": {
            "description": "The list of arguments to pass in to the constructor of the contract being deployed.",
            "type": "array",
            "default": [],
            "items": {}
          },
          "gasConfig": {
            "$ref": "#/components/schemas/GasTransactionConfig"
          },
          "value": {
            "type": "string",
            "description": "Ether balance to send on deployment.",
            "nullable": false
          }
        }
      }
```

However in various tests that input additional properties, and expect an error to be thrown, the test actually passes. 

**To Reproduce**

Run the test beginning on [this line](https://github.com/hyperledger/cacti/blob/957da7c3e1d80068391485a825ba6bb1e68333ac/packages/cactus-plugin-ledger-connector-ethereum/src/test/typescript/integration/geth-contract-deploy-and-invoke-using-json-object-v1.test.ts#L245).

The test should throw an error, but it passes without an error. 

**Expected behavior**

The test should throw an error saying additional properties are forbidden. 

**Logs/Stack traces**

Can help maintainers identify root causes

**Hyperledger Cactus Plugins/Connectors Used**

- Ethereum 

**Additional context**

- This issue was derived from inconsistencies found in [issue 3487](https://github.com/hyperledger/cacti/issues/3487) and [issue 3475](https://github.com/hyperledger/cacti/issues/3475)

**Comment by petermetz** [comment]
@ashnashahgrover Thank you very much, this will help us keep track of the bug!

**Comment by jagpreetsinghsasan** [comment]
From the conversation here: https://github.com/hyperledger/cacti/pull/3496#discussion_r1732296908

**Comment by petermetz** [comment]
@aldousalvarez @jagpreetsinghsasan After any epic rabbit hole diving session I came to realize that this is not supported by the express openapi validator package that we use so please put this back in the backlog.

https://github.com/cdimascio/express-openapi-validator/issues/104

**Comment by jagpreetsinghsasan** [comment]
> @aldousalvarez @jagpreetsinghsasan After any epic rabbit hole diving session I came to realize that this is not supported by the express openapi validator package that we use so please put this back in the backlog.
> 
> [cdimascio/express-openapi-validator#104](https://github.com/cdimascio/express-openapi-validator/issues/104)

Yes, I had a look into this as well. Putting it back in backlog.

**Comment by YYuan02** [comment]
@ashnashahgrover @petermetz Hi, I would like to work on this issue. Can you assign to me?

**Comment by YYuan02** [comment]
@rwalworth Hi, I would like to work on this issue. Can you assign to me?
